ledger/metrics/src/main/scala/com/daml/telemetry/Telemetry.scala

```diff
@@ -30,6 +31,11 @@ abstract class Telemetry(protected val tracer: Tracer) {
     */
   def contextFromMetadata(metadata: Option[jMap[String, String]]): TelemetryContext
 
+  /** TODO
```

Comment:
are you planning to write this doc in this PR?

Reply:
Yes, it's a draft and according to the description, I am going to add it.